internal/pkg/apikey/apikey.go

```diff
@@ -58,6 +58,10 @@ func (k ApiKey) Token() string {
 	return base64.StdEncoding.EncodeToString([]byte(s))
 }
 
+func (k ApiKey) Config() string {
```

Comment:
We can probably find a better name

Reply:
Maybe `Agent()` being this is what gets sent to the agent.